src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: Checking DNS resolution first
+  shell: |
+    dig ipa-ca.{{ ipa_domain }} | grep NOERROR
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 10
+
+- block:
+  - name: Run IPA advise script for server setup for smart cards
+    shell: |
+      kinit admin@{{ ipa_domain.upper() }}
+      ipa-advise config-server-for-smart-card-auth > /tmp/sc_server.sh
+      sh -x /tmp/sc_server.sh {{ sssd_ca_path }} {{ ipa_ca_path }}
```

Comment:
It might be better to place this script elsewhere. Stig mounts /tmp as noexec not sure about fips.

Reply:
How about I use the virt_smartcard_dir variable?  /opt/test_ca should be good right?

Reply:
Yes. That could work. Alternatively I would also consider moving everything under /var if we went to run smartcard tests in image mode in the future.

Reply:
Let's stick with this for now if that's ok.   If we need to run smartcard tests in image mode, I'll submit a PR to update all smart card related paths.